internal/env/config.go

```diff
@@ -50,6 +51,11 @@ type Config struct {
 	AtomicUpgrade      bool     `split_words:"true"`                 // Pass --atomic to `helm upgrade`
 	CleanupOnFail      bool     `envconfig:"cleanup_failed_upgrade"` // Pass --cleanup-on-fail to `helm upgrade`
 	LintStrictly       bool     `split_words:"true"`                 // Pass --strict to `helm lint`
+	EnableV2Conversion bool     `split_words:"true"`                 // Whether or not to use 2to3 convert to migrate Releases from v2 to v3
+	DeleteV2Releases   bool     `split_words:"true"`                 // Pass --delete-v2-releases option for 2to3 convert command
```

Comment:
Should Deleting V2 Releases not be included in this plugin?  Is it better left to the automated checks to determine if tiller is still needed in a namespace.  e.g. If tiller is being removed, remove all v2 releases and create a backup.

Reply:
If the desired behavior is to delete the v2 releases, why have it as an option?

+1+1 On a backup feature.

Reply:
Because this is a public repo so I think it makes more sense to not have highly opinionated defaults. In the way we would run this is by having `delete_v2_releases=true`